### PR TITLE
adding VS hugginface demo

### PR DESCRIPTION
**Pull request by edyoshikun**
PR adds a gif with the demo of VS. 

This PR still needs a bit of polishing.
Before we merge we should change the link to be absolute bsased on specific commit hash or main branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ This repository provides a pipeline for the following.
 ## Virtual staining
 
 ### Demos 
+- [Interactive 2D virtual staining demo of cell nuclei and membrane from labelfree data 🤗 ](https://compmicro-czb-virtualstaining.hf.space)[![Open in Spaces](https://huggingface.co/datasets/huggingface/badges/resolve/main/open-in-hf-spaces-sm-dark.svg)](https://huggingface.co/spaces)
+<center><a href="https://compmicro-czb-virtualstaining.hf.space" target="_blank" rel="noopener noreferrer">
```
Comment (ziw-liu) [inline README.md:21]: Maybe add a line break here before the badge?

Comment (ziw-liu) [reply README.md:21]: Also should the badge link to our space?

Comment (ziw-liu) [reply README.md:21]: The HTML centering for the GIF doesn't work on GitHub.

**Comment by edyoshikun** [comment]
@ziw-liu I wasn't able to center the gif.

**Review by ziw-liu — APPROVED** [review]

**Pull request merged**